.changelog/2836.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
```

Comment:
For update operation the following changelog was defined:
```release-note:enhancement
resource/mongodbatlas_advanced_cluster: Adjusts update operation to support cluster tier auto scaling per shard.
```
WDYT if we align this changelog with a similar format?

Reply:
Done!